notebooks/execute_commands.ipynb

```diff
@@ -0,0 +1,321 @@
+{
+ "cells": [
```

Comment:
How is this notebooks semantically different from notebooks here https://github.com/Azure/azureml-examples/blob/main/sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/pipeline_with_python_function_components.ipynb

Reply:
Conceptually they are similar. This notebook contains the helper functions, right out from the model-factory example we have that's the only difference. Since the DS are comfortable using notebooks, the thought is to provide them with a notebook version of the helper functions and keep it along with the model-factory. This prevents people spending their effort in creating and managing their own versions of the notebook.